notebooks/book1/02/beta_dist_plot.ipynb

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            plt.plot(x, y, p, label="a=%.1f, b=%.1f" % (a, b))
</pre><p>Gap is still not visible in plot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I have added a single space. Will increase the spacing